src/widgetastic_patternfly/__init__.py

```diff
@@ -661,14 +661,15 @@ def currently_selected(self):
             for el
             in self.browser.elements(self.CURRENTLY_SELECTED, parent=self)]
 
-    def select(self, *levels, **kwargs):
+    def select(self, *levels, handle_alert=True, **kwargs):
```

Comment:
I recommend leaving `handle_alert` as a `kwarg` and documenting it like you've done already.

Reply:
I would disagree here, I think its an important kwarg to name. If there's a mismatch of consistency in method declarations and their use of handle_alert has a named kwarg, I think all uses should be updated to be a named kwarg.

Reply:
Then I think `anyway` should be a named kwarg too, otherwise it seems inconsistent.

Reply:
Up to @prichard77, I can include a followup PR if needed.